### ISSUE TITLE
Ready for a new gem version?

### ISSUE THREAD

**Issue by jasnow** [rails/jbuilder#457]
The depreciation warning below got fixed a while back but it's has not be released so users have to pull the gem directly from GitHub to remove that the depreciation warning.
```
DEPRECATION WARNING: Single arity template handlers are deprecated.
Template handlers must now accept two parameters, the view object
and the source for the view object.
```
https://github.com/rails/jbuilder/commit/3481da6b9fcb1785c900c8467a39a25d6f7ed169

**Comment by morgoth** [comment]
@rafaelfranca Since the Rails 6.0.0.rc1 is around the corner, can we have a new release of `jbuilder` to avoid this deprecation warning?

**Comment by connorshea** [comment]
I'd also appreciate a new release :)

**Comment by georgeclaghorn** [comment]
I just released version 2.9.0.
